src/Controls/Geosearch.js

```diff
@@ -17,7 +18,8 @@ export var Geosearch = L.Control.extend({
     L.Util.setOptions(this, options);
 
     if (!options || !options.providers || !options.providers.length) {
-      throw new Error('You must specify at least one provider');
+      options = {};
```

Comment:
i was surprised at first, but it *doesn't* pollute the global namespace to assign a new object to a previously undefined variable.

if @nixta (or any other @Esri/leaflet-contributors) have time to smell test this, that'd be swell